Description: Remove non-linux dependencies
Author: James McCoy <jamessan@debian.org>
Forwarded: not-needed
Last-Update: 2023-07-03
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: alacritty-config/Cargo.toml
===================================================================
--- alacritty-config.orig/Cargo.toml
+++ alacritty-config/Cargo.toml
@@ -29,8 +29,3 @@ version = "1.0.137"
 
 [dependencies.serde_yaml]
 version = "0.8.24"
-
-[target."cfg(target_os = \"macos\")".dependencies.winit]
-version = "0.28.2"
-features = ["serde"]
-default-features = false
